catalog.bom

```diff
@@ -48,24 +109,25 @@ brooklyn.catalog:
           - required
       - name: hyperledger.app.timeout
         description: |
-          The number of seconds the demo app should wait between deploy, invoke, and query steps.
+          The number of seconds the demo app should wait between deploy, invoke, and query
+          steps.
         type: integer
         default: 180
         constraints:
           - required
 
       brooklyn.config:
-        hyperledger.peers.per.location: 4
+        hyperledger.peers.per.location: 3
         hyperledger.pbft.request.timeout: 120
         hyperledger.app.timeout: 30
 
       brooklyn.children:
-      - type: hyperledger-fabric-cluster
-        name: Hyperledger Fabric Cluster
+      - type: hyperledger-fabric-cluster-single-location
+        name: Hyperledger Fabric Cluster (Single-location)
 
-  - id: hyperledger-fabric-cluster
+  - id: hyperledger-fabric-cluster-multi-location
```

Comment:
I don't like the use of `multi-location` for names and descriptions. In Brooklyn a [multi-location](https://brooklyn.apache.org/v/latest/misc/javadoc/org/apache/brooklyn/location/multi/MultiLocation.html) is different to a [fabric](https://brooklyn.apache.org/v/latest/misc/javadoc/org/apache/brooklyn/entity/group/DynamicFabric.html). My preference is to use Hyperledger Fabric fabric, to avoid the confusion from the overloading.